cli/config/configuration-nemu.toml.in

```diff
@@ -10,7 +10,7 @@
 # XXX:   Name: @PROJECT_NAME@
 # XXX:   Type: @PROJECT_TYPE@
 
-[hypervisor.nemu]
+[hypervisor.qemu]
```

Comment:
I presume this was just fixing up a typo, and not matching change is needed in the runtime parser or binary deployment names for instance?

Reply:
It looks like this is in fact by design as the commit mentions:

> use the qemu hypervisor type for configuration template

@egernst - could you add a comment in the file explaining this though please?